src/Files.App/Filesystem/LibraryManager.cs

```diff
@@ -11,78 +11,78 @@
 using Windows.System;
-using Visibility = Microsoft.UI.Xaml.Visibility;
 
 namespace Files.App.Filesystem
 {
 	public class LibraryManager : IDisposable
 	{
-		public EventHandler<NotifyCollectionChangedEventArgs>? DataChanged;
+		private static readonly Lazy<LibraryManager> _lazy = new(() => new LibraryManager());
+
+		private FileSystemWatcher _librariesWatcher;
 
-		private FileSystemWatcher librariesWatcher;
-		private readonly List<LibraryLocationItem> libraries = new();
-		private static readonly Lazy<LibraryManager> lazy = new(() => new LibraryManager());
+		private readonly List<LibraryLocationItem> _libraries;
 
 		public static LibraryManager Default
-			=> lazy.Value;
+			=> _lazy.Value;
 
 		public IReadOnlyList<LibraryLocationItem> Libraries
 		{
 			get
 			{
-				lock (libraries)
-				{
-					return libraries.ToList().AsReadOnly();
-				}
+				lock (_libraries)
+					return _libraries.ToList().AsReadOnly();
 			}
 		}
 
+		public EventHandler<NotifyCollectionChangedEventArgs>? DataChanged;
+
 		public LibraryManager()
 		{
 			InitializeWatcher();
+
+			// Initialize
+			_libraries = new();
 		}
 
 		private void InitializeWatcher()
 		{
-			if (librariesWatcher is not null)
+			if (_librariesWatcher is not null)
 				return;
 
-			librariesWatcher = new FileSystemWatcher
+			_librariesWatcher = new()
 			{
 				Path = ShellLibraryItem.LibrariesPath,
 				Filter = "*" + ShellLibraryItem.EXTENSION,
 				NotifyFilter = NotifyFilters.Attributes | NotifyFilters.LastWrite | NotifyFilters.FileName,
-				IncludeSubdirectories = false,
+				IncludeSubdirectories = false
 			};
 
-			librariesWatcher.Created += OnLibraryChanged;
-			librariesWatcher.Changed += OnLibraryChanged;
-			librariesWatcher.Deleted += OnLibraryChanged;
-			librariesWatcher.Renamed += OnLibraryRenamed;
+			// Start interaction
+			_librariesWatcher.Created += OnLibraryChanged;
+			_librariesWatcher.Changed += OnLibraryChanged;
+			_librariesWatcher.Deleted += OnLibraryChanged;
+			_librariesWatcher.Renamed += OnLibraryRenamed;
 
-			librariesWatcher.EnableRaisingEvents = true;
+			_librariesWatcher.EnableRaisingEvents = true;
 		}
 
 		public static bool IsDefaultLibrary(string libraryFilePath)
 		{
-			// TODO: try to find a better way for this
-			switch (Path.GetFileNameWithoutExtension(libraryFilePath))
+			// TODO: Try to find a better way for this
+			return Path.GetFileNameWithoutExtension(libraryFilePath) switch
 			{
-				case "CameraRoll":
-				case "Documents":
-				case "Music":
-				case "Pictures":
-				case "SavedPictures":
-				case "Videos":
-					return true;
-
-				default:
-					return false;
-			}
+				"CameraRoll" or
+				"Documents" or
+				"Music" or
+				"Pictures" or
+				"SavedPictures" or
+				"Videos" => true,
+				_ => false,
+			};
 		}
 
 		/// <summary>
-		/// Get libraries of the current user with the help of the FullTrust process.
+		/// Gets libraries of the current user with the help of the FullTrust process.
```

Comment:
This comment is outdated, we don't use anymore the Fulltrust process, do we? @yaira2

Reply:
Fixed